code/schemes-numbers.rb

```diff
@@ -60,21 +60,20 @@ def get_one(el, xp)
   end
 end
 puts prefill
-result.sort {|a, b| b <=> a}.each do |value, scheme|
-puts "| %9s | %16s | \\[RFC-XXXX] |"  % [value, scheme]
+result.sort.each do |value, scheme|
+puts "| %17s | %16s | \\[RFC-XXXX] |"  % [value, scheme]
 end
 
 __END__
-
-| CRI value | URI scheme | Reference   |
-|-----------|------------|-------------|
-|        -1 | coap       | \[RFC-XXXX] |
-|        -2 | coaps      | \[RFC-XXXX] |
-|        -3 | http       | \[RFC-XXXX] |
-|        -4 | https      | \[RFC-XXXX] |
-|        -5 | urn        | \[RFC-XXXX] |
-|        -6 | did        | \[RFC-XXXX] |
-|        -7 | coap+tcp   | \[RFC-XXXX] |
-|        -8 | coaps+tcp  | \[RFC-XXXX] |
-|        -9 | coap+ws    | \[RFC-XXXX] |
-|       -10 | coaps+ws   | \[RFC-XXXX] |
+| CRI scheme number | URI scheme | Reference   |
```

Comment:
For consistency with the updated names of the registry columns later on, the names of the columns here should be "CRI scheme number value" and "URI scheme name".

Reply:
I removed the "value" from the IANA registry entry definition -- this seems redundant to me.